app/js/actions/actionTypes.js

```diff
@@ -47,3 +47,6 @@ export const DELETE_ALL_DRAFT_LAB_ORDERS = 'DELETE_ALL_DRAFT_LAB_ORDERS';
 export const SET_ORDER_ACTION = 'SET_ORDER_ACTION';
 export const GET_DATE_SUCCESS = 'GET_DATE_SUCCESS';
 export const GET_DATE_FAILURE = 'GET_DATE_FAILURE';
+export const FETCH_LAB_ORDERABLE_LOADING = 'FETCH_LAB_ORDERABLE_LOADING';
```

Comment:
Should this be `FETCH_LAB_ORDERABLES_LOADING` (the orderable is plural).
What do you think? and the same for success and failure.

Reply:
Yes, you are right... @jomadoye

Reply:
Done.